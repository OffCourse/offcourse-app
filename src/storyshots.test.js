import initStoryshots from "@storybook/addon-storyshots";
import Enzyme, { mount } from "enzyme";
import { createSerializer } from "enzyme-to-json";

import Adapter from "enzyme-adapter-react-16";
Enzyme.configure({ adapter: new Adapter() });

initStoryshots({
  framework: "react",
  renderer: mount,
  snapshotSerializers: [createSerializer()]
});
