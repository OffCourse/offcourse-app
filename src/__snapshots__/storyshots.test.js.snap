// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Button with text 1`] = `
<button
  className="ButtonWrapper-sc-4pr1vz-0 czTlJF"
  disabled={false}
  onClick={[Function]}
  size="SMALL"
  tabIndex={1}
  type="button"
>
  Hello Sir
</button>
`;
